Cargo.lock

```diff
@@ -2907,6 +2922,15 @@ dependencies = [
  "no-std-net",
 ]
 
+[[package]]
```

Comment:
We introduced a second version of same lib, can you make sure we reuse version by adding to workspace deps?

Reply:
Fixed!